src/mkpsxiso/iso.cpp

```diff
@@ -416,7 +409,7 @@ int iso::DirTreeClass::CalculateDirEntryLen() const
 
 		int dataLen = sizeof(cd::ISO_DIR_ENTRY);
 
-		dataLen += GetIDLength(entry.id);
+		dataLen += entry.id.length();

```

Comment:
Here I have big dude, I don't know if no-XA also needs to be rounded to even, for now only the XA isos are rounded.